deb/openmediavault-lvm2/usr/share/openmediavault/engined/rpc/logicalvolumemgmt.inc

```diff
@@ -614,12 +617,74 @@ class OMVRpcServiceLogicalVolumeMgmt extends \OMV\Rpc\ServiceAbstract {
 				"vgname" => $lv->getVGName(),
 				"attributes" => $lv->getAttributes(),
 				"vgattributes" => $lv->getVGAttributes(),
-				"isavailable" => $lv->IsMediaAvailable()
+				"isavailable" => $lv->IsMediaAvailable(),
+				"_used" => $used
 			];
 		}
 		return $result;
 	}
 
+	/**
+	 * Checks /proc/mount for a mount point for a given LVM logical volume path.
+	 *
+	 * @param string $lvPath The full path of the LVM logical volume (can be /dev/dm-0, /dev/vgname/lvname, or /dev/mapper/vgname-lvname).
+	 * @return boolean true if a mount point is found
+	 */
+	private function checkForMountedFilesystem($lvPath) {
```

Comment:
I think it is not necessary to parse the mounts file; a simply database query for `fsname` of `conf.system.filesystem.mountpoint` that uses all device files returned by `getDeviceFileSymlinks` is enough here. Do you agree?